tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -1014,9 +1048,16 @@ class MklFusedBatchNormGradOp : public OpKernel {
       U* weights_data = weights_data_tf;
       T* diff_src_data = static_cast<T*>(diff_src_tensor->flat<T>().data());
       U* diff_weights_data = static_cast<U*>(diff_weights.GetAllocatedBuffer());
+
+      U* res_space_data = nullptr;
```

Comment:
You can make this a constant using the ternary clause.
```C++
const U* res_space_data = (reserved_space)? casting_of_actual_pointer : nullptr;
```
In general, if any variable is not modified in its lifetime, it should be a constant.

Reply:
Done